scripts/build

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+ZIP_NAME="deployment.zip"
+
+pip install --target ./reqs -r sleuth/requirements.txt
+cd reqs
```

Comment:
Totally personal preference. In scripts like these, I prefer a push/pop when you switch directories.